server/controllers/adminController/index.js

```diff
@@ -7,7 +7,9 @@ const AdminUsersController = {
       const { uid } = req.params
       const { value } = await UserModel.update(appDb, uid, req.body)
```

Comment:
should we tweak the `UserModel` to use the same convention as the `ConfiugrationModel`?